src/pages/index.tsx

```diff
@@ -187,14 +190,20 @@ export default function Home() {
           <BsChevronCompactRight onClick={nextSlide} size="40" />
         </div>
 
-        <div className="flex absolute justify-center bottom-4 items-center">
+        <div className="flex absolute space-x-2 justify-center bottom-4 items-center">
           {slidesSetOne.map((slide, slideIndex) => (
             <div
               key={slideIndex}
               onClick={() => goToSlide(slideIndex)}
-              className="text-2xl cursor-pointer"
+              className="text-2xl cursor-pointer flex-row flex"
             >
-              <RxDotFilled />
+              <a
+                className={`h-3 w-3 rounded-full outline-1 outline ${
+                  currentIndex === slideIndex
+                    ? "outline-gray-400 bg-gray-400"
+                    : "outline-gray-400"
```

Comment:
outline-gray-400 is a common class within this className, shouldn't be in ternary

Reply:
addressed.